### ISSUE TITLE
Add support for CRI logs format

### ISSUE THREAD

**Issue by exekias** [elastic/beats#5630]
Container Runtime Interface (CRI) initiative standardizes an API for container implementations. It defines its own format for logs, compatible with Docker `json-file`, but with an alternative implementation (no JSON).

We should update our [Docker prospector](https://github.com/elastic/beats/pull/5402) to support that format, probably we can do autodetection just by checking first chars.

Example log line: 
```
2017-09-12T22:32:21.212861448+00:00 stdout 2017-09-12 22:32:21.212 [INFO][88] table.go 710: Invalidating dataplane cache ipVersion=0x4 reason="refresh timer" table="nat"
```
CRI-o project: https://github.com/kubernetes-incubator/cri-o
CRI-o code writing logs: https://github.com/mrunalp/cri-o/blob/37e282a29b84ca12b594cc55767295f37da848ea/conmon/conmon.c#L290-L407

**Comment by ruflin** [comment]
@exekias Good idea. We should have it available as autodiscovery or config option.

**Comment by exekias** [comment]
Closed by #6181
